### ISSUE TITLE
eventEmitted to receive an event object.

### ISSUE THREAD

**Issue by leonprou** [rkalis/truffle-assertions#24]
**Is your feature request related to a problem?**
For most of the time I use `eventEmitted` filter property to check some props of the emitted event. A more concise way can be just to pass an object to compare it with the event.

**Describe the solution you'd like**
For example I want to check the ERC20 Transfer event. The current form:
```
truffleAssert.eventEmitted(result, 'TokenCreated', (ev) => {
  return ev.from === from && ev.to === to && ev.tokens === tokens
})
```
Can be changed to
```
truffleAssert.eventEmitted(result, 'TokenCreated', {from: from, to: to, tokens: tokens})
```

**Comment by rkalis** [comment]
Hi @leonprou, thanks for opening this issue.

I'm open to the idea. I don't want to start adding other convenience forms, but I think it makes sense to add the one you're suggesting, as I believe that your usage is the most common usage of the library altogether.

At the moment I don't have time to add this, but I can look at it some time in the future. I'm also open to a PR that adds this functionality (including test coverage), and I can provide some assistance to anyone who wants to pick it up.

**Comment by leonprou** [comment]
@rkalis cool, opened a WIP PR https://github.com/rkalis/truffle-assertions/pull/25.

Now I'm performing an `isEqual` with the given event. I wonder if the passed object should  act like a filter object and to ignore the properties that's not given. For example with the current implementation this is not matched:
```
truffleAssert.eventEmitted(result, 'TokenCreated', {from: from})
```

I think that making such examples to match will make this more useful

**Comment by rkalis** [comment]
@leonprou Awesome! I will look at this tomorrow or the weekend :)

**Comment by leonprou** [comment]
merged in https://github.com/rkalis/truffle-assertions/pull/25. @rkalis when you plan to release a new version, can't wait to start using this :)

**Comment by rkalis** [comment]
I will update the docs and publish a new release either tonight or tomorrow morning JST. Thanks again for contributing!

**Comment by rkalis** [comment]
Just published truffle-assertions@0.9.0 which includes your PR 🎉
